python/cudf/cudf/core/dataframe.py

```diff
@@ -2273,7 +2274,7 @@ def iteritems(self):
         for k in self:
             yield (k, self[k])
 
-    @property
+    @property  # type: ignore
```

Comment:
We need to ignore the type on property decorators...?

Reply:
I believe decorators that take arguments aren't supported yet: https://github.com/python/mypy/issues/3157

Reply:
Or it was nested decorators that take arguments...